addons/mail/models/mail_thread.py

```diff
@@ -2951,6 +2951,32 @@ def _message_log_batch(self, bodies, subject=False,
                        for record in self]
         return self.sudo()._message_create(values_list)
 
+    def set_message_pin(self, message_id, pinned):
+        """ (Un)pin a message on the thread and send a notification to the
```

Comment:
Docstring to update as it comes from discuss (there are no members on standard records). Maybe you talked about bus notifications ?